revscoring/languages/german.py

```diff
@@ -0,0 +1,97 @@
+import sys
+
+from .space_delimited import SpaceDelimited
+
+try:
+    from nltk.stem.snowball import SnowballStemmer
+    stemmer = SnowballStemmer("german")
+except ValueError:
+    raise ImportError("Could not load stemmer for {0}. ".format(__name__))
+
+try:
+    from nltk.corpus import stopwords as nltk_stopwords
+    stopwords = set(nltk_stopwords.words('german') + ["a"])
+except LookupError:
+    raise ImportError("Could not load stopwords for {0}. ".format(__name__) +
+                      "You may need to install the nltk 'stopwords' " +
+                      "corpora.  See http://www.nltk.org/data.html")
+
+try:
+    import enchant
+    dictionary = enchant.Dict("de")
+except enchant.errors.DictNotFoundError:
+    raise ImportError("No enchant-compatible dictionary found for 'de'.  " +
+                      "Consider installing 'myspell-de'.")
```

Comment:
$ sudo apt-get install myspell-de
[sudo] password for halfak: 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package myspell-de